applications/portal/backend/api/migrations/0008_auto_20240215_0459.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.2.9 on 2024-02-15 12:59
+
+from django.db import migrations
+from api.utilities.functions import catalog_number_chunked
+
+
+def rechunk_catalog_number(apps, schema_editor):
+    Antibody = apps.get_model('api', 'Antibody')
+    for antibody in Antibody.objects.all():
+        antibody.catalog_num_search = catalog_number_chunked(antibody.catalog_num, antibody.cat_alt)
+        antibody.save()
```

Comment:
this way of saving is too. Probably better do it low level with plain queries. But let's test it first